lib/cmd/fh3/admin/logs/syslogs.js

```diff
@@ -15,11 +15,17 @@ module.exports = {
     'numLogEntries': i18n._("The maximum number of log entries returned. Default is 1000."),
     'orderBy': i18n._("The order of entries, by timestamp, to be displayed. The available options are 'first' or 'last'")
   },
+  'alias' : {
+    'requestId' : 'id',
+    'projects' : 'p',
+    'json' : 'j',
+    0 :'requestId',
+    1 :'projects'
+  },
   'orderingParams': {
     'first': 'asc',
     'last': 'desc'
   },
-  'perm': 'cluster/reseller:read',
```

Comment:
Why is this being removed?
Shouldn't we just check for the same permissions level as on the backend? ( https://github.com/fheng/fh-supercore/blob/4a41bcc0ede97525b4d8e6ca4dd5ca34f180737b/lib/permissions/permissions_map.js#L894 )

Reply:
If we use the perm the command will not available to the user and the result will be "Not found".The idea here exactly changes it and let the customer see the errors according to the roles validation in the backend as it is done for NGUI.

Reply:
Is it not possible to fix fhc so the commands are visible when the user has correct permissions. Removing "perm" seems like a workaround, not a fix.
Does it affect all commands or just the ones that require reseller permissions?

Reply:
@matskiv 

The best approach has the validation of is possible/authorized to do or not in the place that actually has this implementation which is the backend. Not in the NGUI and not in FHC which are client/frontend. So, IMHO it was a mistake adding the perm at the first time here. 

The roles are centralized in the back and if we change it in the back then should reflect across all clients and frontends and not be required to change many places. 

Remove the 'perm' will affect just the commands that had the perm currently which are only for as you can check in the first comment and in the files changes. Is it makes sense?

Reply:
Permissions here were implemented to hide commands that are not accessible by a logged in user, and I think this provides good UX.
I agree that duplicating permissions here is not ideal. But I don't think its likely that we would  need to change a permission for a command.

Reply:
We chat internally about this change. I will close this comments as solved. But please, feel free to reopen if you need.